week-2/D-predicates/exercise.js

```diff
@@ -8,13 +8,13 @@
 // Finish the predicate function to test if the passed number is negative (less than zero)
 
 function isNegative(number) {
- // return typeOf number === "number";
+ // return typeof number === "number";
     return number<0;
 }
 
 // Finish the predicate function to test if the passed number is between 0 and 10
   function isBetweenZeroAnd10(number) {
-  var number = 0
+    var number = (5) 
```

Comment:
This needs to check whether `number > 0` and also `number < 10`, so we need to combine two boolean conditions using the "and" operator (which is called `&&` in Javascript).